src/Pages/Main/Products/Products.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import "./Products.scss";
+
+const Products = ({ product }) => {
+  const [active, setActive] = useState(false);
+  const [productImg, setProductImg] = useState("");
+
+  const { length } = product.options;
+  return (
+    <div
+      onMouseEnter={() => setActive(!active)}
+      onMouseLeave={() => setActive(!active)}
+      className="container"
+    >
+      <Link className="link" to={`detail/${product.productId}`}>
+        <img
+          className="img"
+          alt="nikeCloth"
+          src={!productImg ? `${product.images}` : `${productImg}`}
```

Comment:
```suggestion
          src={!productImg ? product.images : productImg}
```
템플릿 리터럴이 필요 없는 경우입니다

Reply:
😳